packages/metrics/src/Metrics.ts

```diff
@@ -458,25 +511,48 @@ class Metrics extends Utility implements MetricsInterface {
     this.shouldThrowOnEmptyMetrics = true;
   }
 
+  /**
+   * Gets the current number of dimensions stored.
+   * 
+   * @returns the number of dimensions currently stored
+   */
   private getCurrentDimensionsCount(): number {
     return Object.keys(this.dimensions).length + Object.keys(this.defaultDimensions).length;
   }
 
+  /**
+   * Gets the custom config service if it exists.
+   *
+   * @returns the custom config service if it exists, undefined otherwise
+   */
   private getCustomConfigService(): ConfigServiceInterface | undefined {
     return this.customConfigService;
   }
 
+  /**
+   * Gets the environment variables service.
+   *
+   * @returns the environment variables service
+   */
   private getEnvVarsService(): EnvironmentVariablesService {
     return <EnvironmentVariablesService> this.envVarsService;
   }
 
-  private isHigh(resolution: StoredMetric['resolution']): resolution is typeof MetricResolution['High'] {
```

Comment:
I have removed this as it was used both as type guard and runtime check. This is not a major issue in itself, however there are no other usages of this pattern elsewhere in the codebase and since the method that used it has also been simplified the method was now redundant.